grants/__snapshots__/search.test.js.snap

```diff
@@ -4,7 +4,18 @@ exports[`Past Grants Search should return empty results for invalid queries 1`]
 Object {
   "facets": Object {
     "amountAwarded": Array [],
-    "awardDate": Array [],
+    "awardDate": Array [
+      Object {
+        "_id": "last3Months",
+        "label": "Last three months",
+        "value": "2018-07-25|2018-10-25",
```

Comment:
... will this break the tests tomorrow? Not sure if this is something we can handle?

Reply:
... yep: https://travis-ci.org/biglotteryfund/grants-service/builds/446084717?utm_medium=notification&utm_source=email

Reply:
Yeah you either have to delete or replace this value before taking a snapshot (like we do with the email tests in the main app). Or alternatively don't use a single snapshot. Maybe snapshot individual pieces where it makes sense but plain assert against the rest.

Reply:
Could you have a look again – I've just removed the snapshot comparison for that failing test in favour of just testing the number of results directly.